scripts/gitutils.py

```diff
@@ -59,19 +59,20 @@ def git_check_merge(branch1, branch2, git=None):
         git.checkout(branch2)
         is_behind = git.log('{0}..{1}'.format(branch2, branch1),
                             max_count=1).strip()
+        clean_merge = True
         if is_behind:
             try:
                 git.merge('--no-commit', '--no-ff', branch1).strip()
             except sh.ErrorReturnCode_1:
                 # git merge returns 1 when there's a conflict
-                return False
-            else:
-                return True
-            finally:
-                git.merge('--abort')
-                git.checkout(orig_branch)
-        else:
-            return True
+                clean_merge = False
+            git.merge('--abort')
```

Comment:
Nevermind. I see this is only attempting a merge for the purpose of determining if it can be done cleanly, so there is always a merge in progress and it should always be aborted.